packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -22,6 +22,15 @@ message Trace {
 		// enclosed in double quotes, etc.  The value of a private variable is
 		// the empty string.
 		map<string, string> variables_json = 4;
+		// A flag so that it is possible to indicate in the Engine frontend whether/how
+		// variables were modified in the trace.
+		enum PrivateVariableEnforcerType {
```

Comment:
Really digging the trace edition here! Do we want to have a sync'ed PR to the `reports.proto` in monorepo?

Reply:
Yay! I'll be moving this addition to another PR for now to keep this one focused on just adding a new EngineReportingOption. Then I'll get a PR for monorepo ready too (if we decide that including this info is useful!).